pkg/cmd/deadman/deadman.go

```diff
@@ -0,0 +1,93 @@
+package deadman
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	configv1informers "github.com/openshift/client-go/config/informers/externalversions"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"k8s.io/component-base/version"
+
+	configeversionedclient "github.com/openshift/client-go/config/clientset/versioned"
+	"github.com/openshift/library-go/pkg/controller/controllercmd"
+)
+
+type Options struct {
+	controllerContext *controllercmd.ControllerContext
+
+	challengeInterval             time.Duration
+	controllerResponseGracePeriod time.Duration
+}
+
+func NewDeadmanCommand(ctx context.Context) *cobra.Command {
+	o := &Options{
+		challengeInterval:             1 * time.Minute,
+		controllerResponseGracePeriod: 1 * time.Minute,
```

Comment:
@deads2k can you try to make this 11 minutes? it look like some operator have much longer default resync intervals that they should (samples-operator has 10 minutes resync)....

alternative is to make them reconcile immediately after change to their clusteroperator, but in some case that would require understanding how different teams write their operators :-)